spec/features/saml/loa3_sso_spec.rb

```diff
@@ -196,6 +197,35 @@
 
         expect(current_url).to eq saml_authn_request
       end
+
+      it 'provides an option to send another letter' do
+        user = create(:user, :signed_up)
+
+        saml_authn_request = auth_request.create(loa3_with_bundle_saml_settings)
+        visit saml_authn_request
+        sign_in_live_with_2fa(user)
+        click_idv_begin
+        fill_out_idv_form_ok
+        click_idv_continue
+        fill_out_financial_form_ok
+        click_idv_continue
+        click_idv_address_choose_usps
+        click_on t('idv.buttons.mail.send')
+        fill_in :user_password, with: user.password
+        click_submit_default
+        click_acknowledge_personal_key
+        first(:link, t('links.sign_out')).click
+        click_submit_default
```

Comment:
What do you think about putting all these steps between lines 207 and 218 into a well-named method (the method can live in this file), such as `perform_id_verification_with_usps_without_confirming_code_then_sign_out`?